deployment/promotion/promote.py

```diff
@@ -57,21 +97,23 @@
     params['VERTA_CURL_OPTS'] = curl_opts
 else:
     params['VERTA_CURL_OPTS'] = ''
+params['VERTA_CURL_OPTS'] += f' -H @curl_headers'
 
 config = {
     'source': {
-        'model_version_id': atoi(params['VERTA_SOURCE_MODEL_VERSION_ID']),
+        'model_version_id': atoi(params['VERTA_SOURCE_MODEL_VERSION_ID'][2:-2]),
```

Comment:
sus. Going to look at this

Reply:
@nickchappell-verta can you ask why this is here? I don't understand why they'd strip the first and last 2 characters from the ID